SCF-Session-8-Assignment/src/question/Zone.java

```diff
@@ -0,0 +1,36 @@
+package question;
+
+import java.util.ArrayList;
+import java.util.List;
+
+/**
+ * This Class is for the zones in the ZOO. The zones contains cages as a ArrayList in the ZOO.
+ * @author Jugal
+ * Dated 07/24/2019
+ */
+public class Zone {
+	
+	static int zoneId = 0;

```

Comment:
Declare the variables using access modifiers.

Reply:
Given the package access for the variable.